pytket/phir/phirgen_parallel.py

```diff
@@ -289,8 +314,8 @@ def genphir_parallel(
         inp: list of shards
         machine: a QTM machine on which to simulate the circuit
     """
-    max_parallel_tq_gates = len(machine.tq_options)
-    max_parallel_sq_gates = len(machine.sq_options)
+    max_parallel_tq_gates = int(len(machine.tq_options) / 2)
+    max_parallel_sq_gates = int(len(machine.sq_options) / 2)
```

Comment:
Would the following variant using floor division be the same?
```suggestion
    max_parallel_tq_gates = len(machine.tq_options) // 2
    max_parallel_sq_gates = len(machine.sq_options) // 2
```